tests/manifest.jsonld

```diff
@@ -13,6 +13,7 @@
       "@id": "mf:action",
       "@type": "@id"
     },
+    "algorithm": "rdfc:hashAlgorithm",
```

Comment:
Perhaps better to name `hashAlgorithm` to avoid confusion with the main RDFC-1.0 algorithm